frontend/ui/widget/touchmenu.lua

```diff
@@ -734,6 +735,14 @@ function TouchMenu:onMenuHold(item)
                 end
             end)
         end
+    elseif item.help_text or type(item.help_text_func) == "function" then
+        local help_text = item.help_text
+        if item.help_text_func then
+            help_text = item.help_text_func()
```

Comment:
Interesting… some kind of setup wizards in the works? :-P

Reply:
uh, no :) just thought why not (and for similarity with the 2 previous `if ... item.hold_input_func` and `if .. item.hold_callback_func`)

Reply:
Sure, sure. I was just wondering what one might do with it.